index.html

```diff
@@ -5477,10 +5477,20 @@ <h3>Algorithm</h3>
                   <code>xsd:double</code> and its
                   <a>lexical form</a>
                   is a valid <code>xsd:integer</code> or <code>xsd:double</code>
-                  according [[XMLSCHEMA11-2]], set <var>converted value</var>
-                  to the result of converting the
-                  <a>lexical form</a>
-                  to a JSON <a>number</a>.</li>
+                  according to [[XMLSCHEMA11-2]],
+                  <ol>
+                    <li>attempt to convert the <a>lexical form</a> to a <a>JSON number</a>
+                      according to
+                      <a data-cite="RFC8785#name-serialization-of-numbers">
+                        JSON Serialization Scheme (JCS) 3.2.2.3 Serialization of Numbers
+                      </a> procedure;
+                    </li>
+                    <li>
+                      If the conversion is successful, set <var>converted value</var>
+                      to its result.
+                    </li>
```

Comment:
This needs a corresponding "otherwise" to do what 2.5 does, or to explicitly state to use 2.5.

Reply:
What if we remove the "Otherwise" at 2.5?..

Reply:
> What if we remove the "Otherwise" at 2.5?..

That would mean that both 2.4 and 2.8 could be processed for something like `xsd:integer`.

I think we need a 2.4.3.3 which says "Otherwise, set type to the datatype IRI  of value".

We need to keep the "Otherwise, if" in 2.5.

Reply:
Introduced a new point.